linux/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 cmake_minimum_required(VERSION 3.10)
 project(runner LANGUAGES CXX)
 
+set(OPENSSL_USE_STATIC_LIBS OFF)
```

Comment:
Could you maybe still add a comment why this line exist ? Maybe in future we'd otherwise wonder why this line was added, a simple "smth sqlcipher_flutter_libs static linking annoying" comment should do the trick.

Reply:
ok, I will add one

Reply:
Like that comment. Thanks. @krille-chan It's ready for merge.